getting-started/exploring-cloud.md

```diff
@@ -0,0 +1,115 @@
+# Exploring Timescale Cloud
+
+Welcome to Timescale Cloud. Timescale Cloud is a Database as a Service (DBaaS) 
+offering that provides an easy way for you to analyze time-series. Powered 
+by [TimescaleDB][timescale-features], you can create database instances in the 
+cloud and automate many of your most common operational tasks. This allows you 
+to spend more time focusing on your time-series workloads and less time worrying 
+about database management.
+
+Before we start, let's review a few core concepts and phrases:
+
+- **Account**: Your Timescale Cloud account. You can register for a Timescale Cloud account on the [Timescale Cloud signup][sign-up] page.
+- **Project**: An empty project is created for you automatically when you sign-up and are the highest organization level. Projects organize groups of services, and have different billing settings. A way to organize Services in your account and provide access to Services with other users. This is the highest level of access.
```

Comment:
not quite clear to me what `are the highest organization level` refers to, is that privileges?

Reply:
I cleaned up this whole section. I copied it from the Knowledge Base as-is and it didn't make much sense.